master/internal/experiment_filter.go

```diff
@@ -486,3 +484,33 @@ func (e experimentFilter) toSQL(q *bun.SelectQuery,
 	}
 	return q, nil
 }
+
+// validation.loss -> validation_metrics, loss, ”
+// training.loss.min -> avg_metrics, loss, min
+// group_a.value.last -> group_a, value, last
+// group_b.value.a.last -> group_b, value.a, last .
+func parseMetricsName(str string) (string, string, string, error) {
+	if !strings.Contains(str, ".") {
+		return "", "", "", fmt.Errorf("%s is not a valid metrics id", str)
+	}
+	slice := strings.Split(str, ".")
+	if len(slice) < 2 {
+		return "", "", "", fmt.Errorf("%s is not a valid metrics id", str)
+	}
+	metricGroup := slice[0]
+	metricQualifier := slice[len(slice)-1]
+	metricName := strings.Join(slice[1:len(slice)-1], ".")
+	if !slices.Contains(SummaryMetricStatistics, metricQualifier) {
+		metricQualifier = ""
+		metricName = strings.Join(slice[1:], ".")
+	}
```

Comment:
what if i have a validation metric called max and pass `validation.max` for some strange reason.